mm-mock-server/src/mocks/mutations.ts

```diff
@@ -109,6 +109,21 @@ export function mockMutations(serverState: MockServerState) {
             });
             return channelInvitation.id;
         },
+        deleteChannelInvitation: (_: any, args: { deletePhysically: boolean, channelInvitationId: string }) => {
+            const channelInvitation = serverState.channelInvitations.find((e) => e.id == args.channelInvitationId);
+            const currentTime : string = new Date().toISOString();
+            channelInvitation.deletedAt = currentTime;
```

Comment:
Is there already a way to test this in the UI?

Reply:
This is for the Withdraw functionality. And yes you can test it by opening a Sent invitation.